pom.xml

```diff
@@ -15,7 +15,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8
         </project.reporting.outputEncoding>
-        <vaadin.version>21.0.0.alpha6</vaadin.version>
+        <vaadin.version>21.0.0.alpha8</vaadin.version>
```

Comment:
I'm having some trouble with `alpha8`, not sure why but Maven can't download packages; no issues with `alpha10`, we might just want to bump this to the latest.

Reply:
Bumped